src/core/fetch/index.js

```diff
@@ -22,7 +22,7 @@ function loadNested(path, qs, file, next, vm, first) {
 
 function isExternal(url) {
   let match = url.match(
-    /^([^:\/?#]+:)?(?:\/\/([^\/?#]*))?([^?#]+)?(\?[^#]*)?(#.*)?/
+    /^([^:/?#]+:)?(?:\/\/([^/?#]*))?([^?#]+)?(\?[^#]*)?(#.*)?/
```

Comment:
And in https://github.com/docsifyjs/docsify/pull/1227 there is not duplicate code (not two `isExternal` definitions).

Reply:
Same as above: leave as-is or undo this change?

Reply:
Same, I'll fix the conflict